backend/utils/AbsolutePaths.ts

```diff
@@ -0,0 +1,34 @@
+import path from 'path';
+
+let etherpadRoot: string | null = null;
+import findRoot from 'find-root';
+import log4js from 'log4js';
+const absPathLogger = log4js.getLogger('AbsolutePaths');
+
+export const findEtherpadRoot = () => {
+  if (etherpadRoot != null) {
+    return etherpadRoot;
+  }
+
+  const foundRoot = findRoot(__dirname);
+
+  return foundRoot;
```

Comment:
```suggestion
  if (etherpadRoot != null) {
    return etherpadRoot;
  }

  return process.cwd();
```

Here we can simplify by using this it's didn't need any packages and it's work same on our infra (I had test on my machine)

Reply:
True I changed it.

Reply:
Perfect ✨